src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.scss

```diff
@@ -10,3 +10,7 @@
     order: 5;
     flex-basis: 100%;
 }
+
+.bullseye-auto-height{
+    height: auto !important;
```

Comment:
Do we really need this style?
Cause I see the ` <Bullseye style={{ height: 'auto' }}>`